templates/advanced/api/services/Ember.js

```diff
@@ -39,9 +39,9 @@ var Ember = {
 		sideload = sideload || false;
 		var plural = Array.isArray( records ) ? true : false;
 
-		var emberModelIdentity = _.camelCase( model.globalId );
+		var emberModelIdentity = model.globalId;
```

Comment:
This actually needs to stay camelCased for the moment, as otherwise the indexing of a has-many association will be broken. It shouldn't have this impact, but it looks like there's a conceptual flaw with indexing (it assumes that you're using the model name for the inverse property). I'll have to fix it to use the proper `via` property from the association.